src/tests/functional/plugin/cpu/shared_tests_instances/snippets/conv_eltwise.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "snippets/conv_eltwise.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+    const std::vector<InferenceEngine::Precision> netPrecisions = {
```

Comment:
We have both [ov::element](https://github.com/openvinotoolkit/openvino/blob/f45991bd64fd47cb38898355557177d0a68461ab/src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/eltwise.cpp#L44-L48) and [InferenceEngine::Precision](https://github.com/openvinotoolkit/openvino/blob/a79830cb55623ce2654f0685b0ca51b3f412195a/src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/clamp.cpp#L30-L35) in test instantiations, so I don't which is preferable and why.

Anyway, we don't actually use precision parameter in these tests (since we support only fp32 for now), so I removed the unused parameter.

Reply:
Please align the test with OV2.0

Reply:
Ok, Done.